src/shogun/machine/NonParametricMachine.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * This software is distributed under BSD 3-clause license (see LICENSE file).
+ *
+ * Authors: Yuhui Liu
+ */
+
+#ifndef NONPARAMETRCMACHINE_H_
+#define NONPARAMETRCMACHINE_H_
+
+#include <shogun/machine/Machine.h>
+
+namespace shogun
+{
+
+	class NonParametricMachine : public Machine
+	{
+	public:
+		NonParametricMachine()
+		{
+			SG_ADD(
+			    &m_labels, "labels", "labels used in train machine algorithm",
+			    ParameterProperties::READONLY);
```

Comment:
you need to disable this, because it is already present in `Machine`, and then reenable when the whole refactor is done